files/build_templates/sonic_debian_extension.j2

```diff
@@ -353,13 +353,13 @@ echo "procdockerstatsd.service" | sudo tee -a $GENERATED_SERVICE_FILE
 sudo cp $IMAGE_CONFIGS/procdockerstatsd/procdockerstatsd $FILESYSTEM_ROOT/usr/bin/
 
 # Copy systemd timer configuration
-sudo cp $BUILD_TEMPLATES/pcie-mon.timer $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
-sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable pcie-mon.timer
+sudo cp $BUILD_TEMPLATES/pcie-chk.timer $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
```

Comment:
Suggest changing service and timer name to "pcie-check", since it's only two more chars.